src/Command/DrupalUpdate/DrupalOrgClient.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+use Acquia\Cli\Exception\AcquiaCliException;
+use Composer\Semver\Comparator;
+use Exception;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DrupalOrgClient {
+  /**
+   * @var FileSystemUtility
+   */
+  private FileSystemUtility $fileSystemUtility;
+
+  /**
+   * @param FileSystemUtility $fileSystemUtility
+   */
+  protected function setFileSystemUtility(FileSystemUtility $fileSystemUtility): void {
```

Comment:
Getter/setters should be public so that we can call them from outside and inject depedency.

Reply:
Updated code.